pkg/products/rhssouser/reconciler.go

```diff
@@ -47,8 +47,14 @@ var (
 )
 
 const (
-	masterRealmLabelKey   = "sso"
-	masterRealmLabelValue = "master"
+	masterRealmLabelKey         = "sso"
+	masterRealmLabelValue       = "master"
+	developersGroupName         = "rhmi-developers"
+	viewRealmsRoleName          = "view-realms"
```

Comment:
Role name is actually `view-realm`
```
product installation errors : failed installation of rhssouser: failed to
    reconcile rhmi-developers group: view-realms role not found as available
```

Reply:
Oops I'll update it in a moment, thanks for pointing it out :grimacing: